src/Databases/Sqlite.php

```diff
@@ -18,8 +18,16 @@ public function dumpToFile(string $dumpFile): void
 
     public function getDumpCommand(string $dumpFile): string
     {
-        $includeTables = rtrim(' '.implode(' ', $this->includeTables));
-
+        $includeTables = rtrim(' ' . implode(' ', $this->includeTables));
+        if (empty($includeTables) && ! empty($this->excludeTables)) {
+            $db = new \SQLite3($this->dbName);
+            $query = $db->query("SELECT name FROM sqlite_schema WHERE type='table' AND name NOT LIKE 'sqlite_%';");
```

Comment:
Could you extract this to dedicated function with a good names that explains what it does?